### ISSUE TITLE
[JENKINS-28867] Triggering Manual Steps doesn't work in combination with cloudbees-folder

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/build-pipeline-plugin#314]
<p>We already had a Build Pipeline overview of existing jobs with downstream triggers.</p>

<p>Afterwards, the Folder plugin was installed and all relevant jobs were moved to the newly created  folder.</p>

<p>Jobs which use the normal Trigger Parametrized build on other project  work, but then it's broken for jobs using the Build Other Project (Manual Step).</p>

<p>When looking at the Build Pipeline view, triggering the Manual Job results in a progressbar running, but then turns back to the state of a job which hasn't been run before (at this point a stack trace with exceptions is being thrown)</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/carroarmato0.png" title="carroarmato0's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=carroarmato0">carroarmato0</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-28867" target="_blank">Triggering Manual Steps doesn't work in combination with cloudbees-folder</a></i></summary>
<i><ul>
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>component(s)</b>: build-pipeline-plugin
<li><b>label(s)</b>: exception, folders, job
<li><b>resolution</b>: Duplicate
<li><b>resolved</b>: 2016-03-11T20:49:18+00:00
<li><b>votes</b>: 2
<li><b>watchers</b>: 5
<li><b>imported</b>: 20260107-091649
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>We already had a Build Pipeline overview of existing jobs with downstream triggers.</p>

<p>Afterwards, the Folder plugin was installed and all relevant jobs were moved to the newly created  folder.</p>

<p>Jobs which use the normal Trigger Parametrized build on other project  work, but then it's broken for jobs using the Build Other Project (Manual Step).</p>

<p>When looking at the Build Pipeline view, triggering the Manual Job results in a progressbar running, but then turns back to the state of a job which hasn't been run before (at this point a stack trace with exceptions is being thrown)</p></pre>
</details>
</details>
<details><summary><i>environment</i></summary>

```
jenkins-war:1.616<br/>
JRE 1.7.0 IBM J9 2.6 Linux amd64-64<br/>
SLES 11 SP3
```
</details>
<details><summary><i>1 attachment</i></summary>

- [Trace.txt](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/29943/Trace.txt)
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>kassen:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-28867?focusedId=230411&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-230411">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Front end error:</p>

<div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
<pre class="code-java">POST http:<span class="code-comment">//server:8080/$stapler/bound/f80b9adc-fee6-43a3-a6e4-069d87dea705/triggerManualBuild 500 (Server Error)
</span>Ajax.Request.<span class="code-object">Class</span>.create.request	@	prototype.js:1585
Ajax.Request.<span class="code-object">Class</span>.create.initialize	@	prototype.js:1550
<span class="code-object">Object</span>.extend.__method	@	prototype.js:452
klass	@	prototype.js:101
proxy.(anonymous function)	@	bind.js:51
BuildPipeline.triggerBuild	@	build-pipeline.js:80
onclick	@	(index):126
</pre>
</div></div>
</pre>
</details>

</ul>
</details>

<p>Front end error:</p>


<pre>
POST http:<span class="code-comment">//server:8080/$stapler/bound/f80b9adc-fee6-43a3-a6e4-069d87dea705/triggerManualBuild 500 (Server Error)
</span>Ajax.Request.<span class="code-object">Class</span>.create.request	@	prototype.js:1585
Ajax.Request.<span class="code-object">Class</span>.create.initialize	@	prototype.js:1550
<span class="code-object">Object</span>.extend.__method	@	prototype.js:452
klass	@	prototype.js:101
proxy.(anonymous function)	@	bind.js:51
BuildPipeline.triggerBuild	@	build-pipeline.js:80
onclick	@	(index):126
</pre>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/dserodio.png" title="dserodio's avatar" /> dserodio:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-28867?focusedId=250311&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-250311">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>The workaround suggested by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kassen" class="user-hover" rel="kassen">kassen</a> does work, even thou the <tt>Organization/</tt> prefix looks like a relative path (since it does not start with a <tt>/</tt>) but is in fact absolute.</p>
</pre>
</details>

</ul>
</details>

<p>The workaround suggested by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kassen" class="user-hover" rel="kassen">kassen</a> does work, even thou the <tt>Organization/</tt> prefix looks like a relative path (since it does not start with a <tt>/</tt>) but is in fact absolute.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/dserodio.png" title="dserodio's avatar" /> dserodio:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-28867?focusedId=250312&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-250312">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Closing as duplicate of <a href="https://issues.jenkins.io/browse/JENKINS-23532" title="Manual trigger execution causes TriggerException" class="issue-link" data-issue-key="JENKINS-23532"><del>JENKINS-23532</del></a> since that one is older and has more information than this one.</p>
</pre>
</details>

</ul>
</details>

<p>Closing as duplicate of <a href="https://issue-redirect.jenkins.io/issue/23532" title="Manual trigger execution causes TriggerException" class="issue-link" data-issue-key="<a class="jira-plain-text-key" href="https://issue-redirect.jenkins.io/issue/23532">JENKINS-23532</a>"><del>JENKINS-23532</del></a> since that one is older and has more information than this one.</p>

**Comment by jenkins-infra-bot** [comment]
<i>[Original `depends_on` from Jira: <a class="jira-relationship" href="https://issue-redirect.jenkins.io/issue/23532">JENKINS-23532</a>]</i>
